planning/obstacle_cruise_planner/src/node.cpp

```diff
@@ -1375,50 +1375,37 @@ void ObstacleCruisePlannerNode::checkConsistency(
   const rclcpp::Time & current_time, const PredictedObjects & predicted_objects,
   std::vector<StopObstacle> & stop_obstacles)
 {
-  const auto current_closest_stop_obstacle =
-    obstacle_cruise_utils::getClosestStopObstacle(stop_obstacles);
-
-  if (!prev_closest_stop_obstacle_ptr_) {
-    if (current_closest_stop_obstacle) {
-      prev_closest_stop_obstacle_ptr_ =
-        std::make_shared<StopObstacle>(*current_closest_stop_obstacle);
+  static std::vector<StopObstacle> prev_closest_stop_obstacles{};
```

Comment:
Any reason to use static variables?
I actually am not 100% confident, but I feel they should be avoided.
https://stackoverflow.com/questions/7026507/why-are-static-variables-considered-evil